site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md

```diff
@@ -0,0 +1,119 @@
+- [The Svelte Handbook](https://thevalleyofcode.com/svelte/) by The Valley of Code
+- [Test Svelte Component Using Vitest & Playwright](https://davipon.hashnode.dev/test-svelte-component-using-vitest-playwright) by David Peng
+- [Transitional Apps with Phoenix and Svelte](https://nathancahill.com/phoenix-svelte) by Nathan Cahill
+
+_Tech Demos_
+- [Bringing the best GraphQL experience to Svelte](https://www.the-guild.dev/blog/houdini-and-kitql) by The Guild
+- [Style your Svelte website faster with Stylify CSS](https://stylifycss.com/blog/style-your-svelte-website-faster-with-stylify-css/) by Stylify
+- [Revamped Auth Helpers for Supabase (with SvelteKit support)](https://supabase.com/blog/2022/07/13/supabase-auth-helpers-with-sveltekit-support) by Supabase
+
+
+**Libraries, Tools & Components**
+- [Lucia](https://github.com/pilcrowOnPaper/lucia-sveltekit) is a simple, JWT based authentication library for SvelteKit that connects your SvelteKit app with your database
+- [Skeleton](https://github.com/Brain-Bones/skeleton) is a UI component library for use with Svelte + Tailwind
+- [pass-composer](https://pass-composer.vercel.app/) helps you compose your postprocessing passes for threlte scenes
+- [@crikey/stores-*](https://www.npmjs.com/package/@crikey/stores-base) is a collection of 8 libraries to extend Svelte stores for common use-cases
+- [Svelte Chrome Storage](https://github.com/shaun-wild/svelte-chrome-storage) is a lightweight abstraction between Svelte stores and Chrome extension storage
+- [Svelte Schema Form](https://github.com/restspace/svelte-schema-form) is a form generator for JSON schema
+- [svelte-gesture](https://github.com/wobsoriano/svelte-gesture) is a library that lets you bind richer mouse and touch events to any component or view
+- [Snap Layout](https://github.com/ThaUnknown/snap-layout) is a window resizing module for PWAs
+- [svelte-adapter-bun](https://github.com/gornostay25/svelte-adapter-bun) is an adapter for SvelteKit apps that generates a standalone Bun server
+- [json2dir](https://www.npmjs.com/package/json2dir) converts JSON objects into directory trees
+- [Svelte Command Palette](https://github.com/rohitpotato/svelte-command-palette) is a drop-in command palette component
+- [svelte-use-drop-outside](https://github.com/untemps/svelte-use-drop-outside) is a Svelte action to drop an element outside an area
+- [PowerTable](https://github.com/muonw/powertable) is a JavaScript component that turns JSON data into an interactive HTML table
+- [svelte-slides](https://github.com/rajasegar/svelte-slides) is a slide show template for Svelte using Reveal.js
+- [Svelte Theme Light](https://marketplace.visualstudio.com/items?itemName=webmaek.svelte-theme-light) is a Visual Studio Code theme based on the Svelte REPL
+
```

Comment:
- [svelte-simple-store](https://www.npmjs.com/package/simple-store-svelte) A drop in replacement for svelte/store which uses setters/getters to write/read the store's value.
- [universal-title-bar](https://www.npmjs.com/package/universal-title-bar) Universal title bar module, which works the same both on electron and in PWA's and can be imported as a .svelte module or as a web component
- [snap-layout](https://www.npmjs.com/package/snap-layout) Windows 11's snap layout feature for webapps using svelte, can be imported as a svelte component or as a web component
-

Reply:
I try to avoid more than one bullet from the same person. I think I'll combine `universal-title-bar` into the Snap Layout bullet and we can mentioned `svelte-simple-store` next month :)

Reply:
they are separate modules, which do different things, there were no issues listing multiple modules from a single person before